ZeeHomeScreen/Classes/Components/SectionComposite/SectionCompositeViewController.swift

```diff
@@ -274,27 +274,43 @@ import Zee5CoreSDK
         }
         toggle.setButtonsTitles(buttonsTitles: ["nav_livetv".localized(hashMap: [:]), "EPG_Header_ChannelGuide_Text".localized(hashMap: [:])])
         toggle.changeToIndex = { [weak self] index in
-            self?.collectionView?.isHidden = index == 0 ? false : true
-            self?.epgContentView?.isHidden = index == 0 ? true : false
+            switch index {
+            case 0:
+                self?.collectionView?.isHidden = false
+                self?.epgContentView?.isHidden = true
+
+            case 1:
+                self?.prepareEPGViewIfNeeded()
+                self?.collectionView?.isHidden = true
+                self?.epgContentView?.isHidden = false
+
+            default: ()
+            }
         }
         toggle.isHidden = false
     }
     
-    private func prepareEPGView() {
+    private func prepareEPGViewIfNeeded() {
+        guard epgContentView?.subviews.isEmpty == true else { return }
 
         let screenID = screenConfiguration?.epgScreenID
 
-        guard let viewController = GARootHelper.uiBuilderScreen(by: screenID!,
-                                                                model: nil,
-                                                                fromURLScheme:nil) else {
-                                                                    APLoggerError("Can not create view controller with ScreenType: \(screenID!) isn't found")
-                                                                    return
+        guard let viewController = GARootHelper.uiBuilderScreen(
+            by: screenID!,
+            model: nil,
+            fromURLScheme: nil
+        ) as? GAScreenPluginGenericViewController else {
+            APLoggerError("Can not create view controller with ScreenType: \(screenID!) isn't found")
+            return
         }
-        (viewController as! GAScreenPluginGenericViewController).isContainerViewController = true
+
+        viewController.isContainerViewController = true
         addChild(viewController)
 
-        epgContentView!.addSubview(viewController.view)
+        epgContentView?.addSubview(viewController.view)
         viewController.view.matchParent()
+
+        viewController.didMove(toParent: self)
```

Comment:
It was missing and is required according to Apple => https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html